packages/utils/src/abi_encoder/evm_data_types/pointer.ts

```diff
@@ -11,7 +11,11 @@ export class PointerDataType extends AbstractPointerDataType {
         super(dataItem, dataTypeFactory, destDataType, parentDataType);
     }
 
-    public getSignature(): string {
-        return this._destination.getSignature();
+    public getSignatureType(): string {
+        return this._destination.getSignature(false);
+    }
+
+    public getSignature(detailed?: boolean): string {
```

Comment:
```suggestion
    public getSignature(isDetailed?: boolean): string {
```

Stick to boolean naming conventions.

Reply:
Let's investigate why was this not caught by linter rule

Reply:
I suspect it's because it's a param name and not a variable. We should improve our linter rule.